ACM_General/core/templates/core/base.html

```diff
@@ -1,85 +1,52 @@
+      <link rel="manifest" href="{% static "/home/img/manifest.json" %}">
+      <link rel="mask-icon"
+            href="{% static "/home/img/safari-pinned-tab.svg" %}"
+            color="#5bbad5">
+      <meta name="theme-color" content="#ffffff">
+      <!--[if lt IE 9]>
 				<script src="lib/js/html5shiv.js"></script>
-				<![endif]-->
-            {% endblock %}
-
-            {% block head_charset %}
-                <meta charset="utf-8" />
-            {% endblock %}
-
-            {% block head_keywords %}
-                <meta name="keywords" content="acmmst, mstacm, ACM General, ACM" />
-            {% endblock %}
-
-            {% block head_description %}
-                <meta name="description" content="ACM-General Site" />
-            {% endblock %}
-
-            {% block head_author %}
-                <meta name="author" content="ACM-MST SIG.com" />
-            {% endblock %}
-
-            {% block head_robots %}
-                <meta name="robots" content="NONE,NOARCHIVE" />
-            {% endblock %}
-
-            {% block head_base_stylesheets %}
-            {% endblock %}
-
-            {% block head_stylesheets %}
-            {% endblock %}
-
-            {% block head_javascript %}
-            {% endblock %}
-
-            {% block head_extra-descriptors %}
-            {% endblock %}
-
-            {% block head_fonts %}
-            {% endblock %}
-
-            {% block head_title %} 
-                <title> ACM | General </title>
-            {% endblock %}
-        {% endblock %}
-
-        {% block head_main %}
-        {% endblock %}
-    </head>
-
-    <body>
-        <!--[if lt IE 8]>
-                <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
-        <![endif]-->
-
-        {% block body_header %}
-        {% endblock %}
-
-        {% block body_content %}
-        {% endblock %}
-
-        {% block body_footer %}
-        {% endblock %}
-
-	{% block body_scripts %}
-	{% endblock %}
-    </body>
+				<![endif]-->{% endblock %}
+    {% block head_charset %}
+      <meta charset="utf-8"/>{% endblock %}
+    {% block head_keywords %}
+      <meta name="keywords" content="acmmst, mstacm, ACM General, ACM"/>
+    {% endblock %}
+    {% block head_description %}
+      <meta name="description" content="ACM-General Site"/>{% endblock %}
+    {% block head_author %}
+      <meta name="author" content="ACM-MST SIG.com"/>{% endblock %}
+    {% block head_robots %}
+      <meta name="robots" content="NONE,NOARCHIVE"/>{% endblock %}
+    {% block head_base_stylesheets %}{% endblock %}
+    {% block head_stylesheets %}{% endblock %}
+    {% block head_javascript %}{% endblock %}
+    {% block head_extra-descriptors %}{% endblock %}
+    {% block head_fonts %}{% endblock %}
+    {% block head_title %}
+      <title> ACM | General </title>{% endblock %}{% endblock %}
```

Comment:
Honestly I don't like it either, from our end it is horrid, and I think it is less important to have it look good on the clients end if it looks bad to the developers, I will remove some of the de-whitespacing I did. It is very strange how Django adds up to 2 lines of whitespace per tag.